lib/extensions.js

```diff
@@ -63,6 +64,8 @@ function getBinaryName() {
     binaryName = flags['--sass-binary-name'];
   } else if (process.env.SASS_BINARY_NAME) {
     binaryName = process.env.SASS_BINARY_NAME;
+  } else if (process.env.npm_config_sass_binary_name) {
+    binaryName = process.env.npm_config_sass_binary_name;
```

Comment:
> This mechanism works bidirectional and npm exposes all configuration parameter into this env namespace

Do you have a link to the documentation for this. I was under the impression it was a one way street. ENV overwrote .npmrc but not the other way around. Happy to be wrong.